core/config/locales/en.yml

```diff
@@ -711,6 +711,7 @@ en:
     add_action_of_type: Add action of type
     add_country: Add Country
     add_coupon_code: Add Coupon Code
+    add_line_item: Add Item
```

Comment:
There is a rake task in `solidus_i18n` for this purpose. This question is way outside the scope of this PR.